### DISCUSSION TITLE
Reorganizing repository files

### DISCUSSION THREAD

**Discussion by Minabsapi** [Jenna1337/NDSROM#5] [Ideas]
Upon giving a quick look at the repository, here are some ideas I got to tidy it a bit:
- First, get rid of those HTML files. These are saved pages that do not belong to you. Remove URL files as well, which were already useless because you already had the pages, but even without them they're of no use for people who aren't working on Windows. Instead, you should have a "docs" folder at the root of your repository and put every documentation (internal or external) there. And so instead of raw pages you should have a markdown document named whatever you want containing those links in. The best would be to create a wiki and add an "External references" page but eh, let's keep things simple for now
- Second, as I already stated in my issue (#1) that brought this conversation, your README doesn't help much. It's a good start, but some instructions on how to get the project working would be necessary
- ...But that could be figured out at least if the entrypoint wasn't buried in some folder. I suppose it is the [convertNDS.bat](/Jenna1337/NDSROM/blob/master/utils/convertNDS.bat) file. You should put it at the root of your repository and eventually give a more obvious name to it (though this one is good and descriptive enough)
- (By the way, Windows users aren't the only ones who'd be interested in this project, so some adaptations for Linux and Mac users would be great. But if you're not well versed in that I may contribute with a translation of your conversion script and other things, when I figure out this repository a little better)
- Locating the entrypoint is good, but I'm curious about the other files. There's some Python scripts, with one mentioned in your script -- for the love of God, *please* don't hardcode paths; not everyone has Python 2.7 and if so, located at the specified folder! -- but I'm curious about the files in Java, C, C++... Even Rust! Are those all dependencies? I start to think this project is just about a script and "stolen" resources and software + sources. Am I wrong? If not, you might want to rethink how you're proceeding; I don't think you can legally keep source files from other projects without mentioning them, or even keep them at all in your repository. Perhaps you could even wrap it all up with a script that has sources and references in comments, instructions to download dependencies or even automated downloads with some cURL magic, all in a single Github snippet?

**Comment by Jenna1337** [comment]
Honestly, most of the folders in utils should probably be git submodules. 
As for the rest of the stuff you mentioned, I'll try to fix them soon

**Reply by Jenna1337** [reply]
There, I think I cleaned it up a bit.

**Reply by Minabsapi** [reply]
It's indeed way better! Now I see more clearly that's a full project using dependencies, please forget my last point (which wasn't meant to be harsh in any way, but that's the feeling I get re-reading this)
